services/efs-guardian/tests/unit/conftest.py

```diff
@@ -0,0 +1,69 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+from pathlib import Path
+from typing import AsyncIterator
+
+import httpx
+import pytest
+from asgi_lifespan import LifespanManager
+from faker import Faker
+from fastapi import FastAPI
+from pytest import MonkeyPatch
+from pytest_simcore.helpers.utils_envs import EnvVarsDict, setenvs_from_dict
+from simcore_service_efs_guardian.core.application import create_app
+from simcore_service_efs_guardian.core.settings import ApplicationSettings
+
+pytest_plugins = [
+    "pytest_simcore.cli_runner",
+    "pytest_simcore.environment_configs",
+    "pytest_simcore.repository_paths",
+]
+
+
+@pytest.fixture(scope="session")
+def project_slug_dir(osparc_simcore_root_dir: Path) -> Path:
+    # fixtures in pytest_simcore.environs
+    service_folder = osparc_simcore_root_dir / "services" / "resource-usage-tracker"
+    assert service_folder.exists()
+    assert any(service_folder.glob("src/simcore_service_resource_usage_tracker"))
+    return service_folder
+
+
+@pytest.fixture
+def app_environment(monkeypatch: MonkeyPatch, faker: Faker) -> EnvVarsDict:
```

Comment:
OK, let's modify these fixture. It hsould call first `env_devel_dict` which returns a `dict` with the env-vars in `.env-devel` file. Then just use `setenvs_from_dict` to mock the environment 
```python
@pytest.fixture
def app_environment(monkeypatch: MonkeyPatch, env_devel_dict: EnvVarsDict) -> EnvVarsDict:
      return setenvs_from_dict(monkeypatch, {**env_devel_dict})

```